### ISSUE TITLE
IO Max, Read and Write meaning

### ISSUE THREAD

**Issue by 4815162342lost** [dalibo/pg_activity#180]
Hello, i do not fully understand this meaning:
```
  Mem.:    4.50% -     1.43G/31.42G   | IO Max:    27580/s
  Swap:    0.40% -    33.59M/7.45G    | Read :      1.15M/s -    294/s
  Load:    2.05 2.20 2.15             | Write:      7.91K/s -      1/s
```

IO MAX -- is it "max IOPS (Read + Write) for all time?"
Read: 294/s -- is it mean 294 IOPS for Read Operations?
Write: 1/s is it mean 1 IOPS for Write Operations?

Could you please clarify and update documentation?

**Comment by dlax** [comment]
Explanations can be found in #213; I agree we should document them.

**Comment by dlax** [comment]
Fixed by #215, information will be available in the man page.
